nest-city-account/src/user-verification/utils/subservice/verification.subservice.ts

```diff
@@ -57,7 +57,13 @@ export class VerificationSubservice {
             message: { message: 'ok' },
           }
         }
-        const identityCardMagproxy = document.jednoznacnyIdentifikator.split(' ') //Identity card numbers are in registry in format "0000 xx" and users from identity card enters "xx0000"
+
+        // Some identity card numbers are in format "000000 XX" in registry, but users enter identity card as "XX000000"
+        const identityCardMagproxy = document.jednoznacnyIdentifikator.trim().split(' ')
+        if (identityCardMagproxy.length !== 2) {
+          continue // Skip if the format is not "000000 XX" as expected
+        }
+
         if (identityCardMagproxy[1] + identityCardMagproxy[0] === identityCard) {
```

Comment:
I would just merge it with the next condition statement:
```
if (identityCardMagproxy.length === 2 && identityCardMagproxy[1] + identityCardMagproxy[0] === identityCard) {
...
}
```

WDYT?

Reply:
`continue` is really unusual in JS world

Context if you want to dig in this:
https://stackoverflow.com/questions/11728757/why-are-continue-statements-bad-in-javascript
https://eslint.org/docs/latest/rules/no-continue